spec/mailers/notification_mailer_spec.rb

```diff
@@ -220,5 +220,34 @@
         expect(mail.body).to match(contributions)
       end
     end
+
+    context 'when a registration email is sent again through admin' do
+      let(:user) { create(:user, name: "Art Vandelay") }
```

Comment:
on the test environment the mailer does not like names with any form of accentuation such as `´`, `^`, etc.